whois-api/src/main/java/net/ripe/db/whois/api/rest/WhoisRestService.java

```diff
@@ -427,4 +433,19 @@ void setDryRun(final UpdateContext updateContext, final String dryRun) {
             updateContext.dryRun();
         }
     }
+
+    private void addBasic(final HttpServletRequest request, final List<String> passwords){
+        final String authHeader = request.getHeader(HttpHeaders.AUTHORIZATION);
+        if (authHeader== null || !authHeader.toUpperCase().startsWith(BASIC_AUTH)){
+            return;
+        }
+        if (!HttpScheme.HTTPS.is(request.getHeader(HttpHeaders.X_FORWARDED_PROTO))){
```

Comment:
original query password is  a list of passwords and here do we just allow one password or did it support multiple passwords?

Reply:
we support to combine passwords and basic auth and you can still add more than one password in password parameter. I created tests for this

Reply:
Does basic Auth supports multiple password? Can you point to test case ? Reason I am asking is password as query parameter is list of password which means user can specify multiple password (may be they need to authenticate with multiple passwords)

Reply:
I think that make not sense to create a test for it (this test will always throw an error 401 doesn't matter if we change the code or not)...you can not add multiple basic auth as far as I know. 
I will using manual tests but I think that this is not possible

https://stackoverflow.com/questions/29282578/multiple-http-authorization-headers
No, it's not possible. See the syntax definition in http://greenbytes.de/tech/webdav/rfc7235.html#header.authorization

Reply:
This is an drawback to basic authentication, and an advantage of using `password` as sometimes you need to authenticate as multiple mntners, although it's an edge case - we should check if anyone actually does that